### PR TITLE
Task #6

### PR DESCRIPTION
**Pull request by erikAlberto**
add the files environment.bat, vagrantfile and docker-compose.

### CODE REVIEW

File: vagrant/Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    config.vm.define 'linux-server-4' do |s|
+        s.vm.provision "docker"
+        s.vm.provision "docker_compose", compose_version: '1.23.1'
+        s.vm.provider :openstack do |os, override|
+            os.server_name = 'AT07ERVA04'
+            os.flavor = 'f2.desktop.tiny'
+            override.vm.synced_folder '.', '/vagrant', disabled: true
+        end
+    end
+end
```
Comment (fercho-ayala) [inline vagrant/Vagrantfile]: Add a new line at end of file.

File: vagrant/docker/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      - "50000:50000"
+    volumes:
+      - ./jenkins:/var/jenkins_home
+  jenkins_slave:
+    image: "registry.jala.info/devops/jenkins/jnlp-slave:3.27-1-alpine"
+    environment:
+      - JENKINS_URL=http://jenkins_server:50000
+      - JENKINS_AGENT_WORKDIR=/var/jenkins_home
+      - JENKINS_SECRET
+      - JENKINS_AGENT_NAME=slave2
```
Comment (fercho-ayala) [inline vagrant/docker/docker-compose.yml]: Add a new line at end of file.

File: vagrant/environment.bat [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+set OS_AUTH_URL=http://10.28.3.240:5000/v3
+set OS_IDENTITY_API_VERSION=3
+set OS_TENANT_NAME=1fb71c6e155f4f1993e867da5f4dce55
+set OS_PROJECT_NAME=fj_erikvargas_at07-desktop
+set OS_USERNAME=ErikVargas
+set OS_DOMAIN_NAME=fundacion-jala
+set OS_FLAVOR=f2.desktop.tiny
+set OS_IMAGE=ubuntu-14.04-amd64-server_18062018
+set OS_KEY_PAIR_NAME=test
+set OS_REGION_NAME=bootstack-jalasoft
```
Comment (fercho-ayala) [inline vagrant/environment.bat]: Add a new line at end of file.

File: vagrant/Vagrantfile [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        os.domain_name = ENV['OS_DOMAIN_NAME']
+        os.username = ENV['OS_USERNAME']
+        os.password = ENV['OS_PASSWORD']
+        os.tenant_name = ENV['OS_TENANT_NAME']
+        os.project_name = ENV['OS_PROJECT_NAME']
+        os.keypair_name = ENV['OS_KEY_PAIR_NAME']
+        os.region = ENV['OS_REGION_NAME']
+        os.image = ENV['OS_IMAGE']
+    end
+    config.vm.define 'linux-server-4' do |s|
```
Comment (fercho-ayala) [inline vagrant/Vagrantfile]: Why are you using 4 ?

Comment (erikAlberto) [reply vagrant/Vagrantfile]: because I already have created a linux-server-1 and I can not create a new with same name.

**Pull request merged**